lms/djangoapps/bulk_email/models.py

```diff
@@ -270,36 +270,16 @@ def create(
         """
         Create an instance of CourseEmail.
         """
+        # deferred import to prevent a circular import issue
+        from lms.djangoapps.bulk_email.api import determine_targets_for_course_email
```

Comment:
Importing this at the top of the file results in a circular import issue. I've seen this method of deferred imports used before in our codebase so I considered it acceptable as prior art. Let me know if you disagree.